pkg/authorization/registry/clusterpolicy/etcd/etcd.go

```diff
@@ -21,11 +22,12 @@ type REST struct {
 }
 
 // NewStorage returns a RESTStorage object that will work against nodes.
-func NewStorage(s storage.Interface) *REST {
+func NewStorage(optsGetter restoptions.Getter) *REST {
+
 	store := &etcdgeneric.Etcd{
 		NewFunc:           func() runtime.Object { return &authorizationapi.ClusterPolicy{} },
 		NewListFunc:       func() runtime.Object { return &authorizationapi.ClusterPolicyList{} },
-		QualifiedResource: authorizationapi.Resource("clusterpolicy"),
+		QualifiedResource: authorizationapi.Resource("clusterpolicies"),
```

Comment:
Most of the time singular is correct: create, delete, update, get (and their multiple flavors) all singular.  Only list is plural.  I'd prefer the common case look right.